app/views/accounts/connected_accounts/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1 class="mt0">
+  <%= t('headings.account.connected_accounts') %>
+</h1>
+
+<p>
+  <%= t('account.connected_apps.description') %>
+</p>
+
+<div class="mb3 card profile-info-box border-bottom border-light-blue">
+  <div class="border-bottom border-light-blue">
+    <% @view_model.connected_apps.each do |identity| %>
+      <%= render identity.connected_app_partial, identity: identity %>
+    <% end %>
```

Comment:
From what I can tell, it was this way as well previously, but I think in the case that the user has no connected accounts, we could be doing better to indicate this on the page:

![no connected accounts](https://user-images.githubusercontent.com/1779930/93483360-bc0ad780-f8ce-11ea-9b8e-b5220698642c.png)

I can create a ticket if you think it makes sense.

Reply:
yeah, it was

I think it would be easier enough to do a lot better. I'm hesitant to put any language at this point, but is it worth not showing the white box at least?